app/v1/models/request/TaxYear.scala

```diff
@@ -26,7 +26,7 @@ final case class TaxYear private (private val value: String) {
     */
   val year: Int = value.toInt
 
-  /** The tax year in MTD (vendor-facing) format, e.g. "2024".
+  /** The tax year in MTD (vendor-facing) format, e.g. "2023-24".
```

Comment:
Is the `asDownstream` comment also incorrect? Would you be able to update that too whilst you're here?

Reply:
fixed